worlds/kh2/__init__.py

```diff
@@ -400,6 +398,8 @@ def goofy_pre_fill(self):
             # plando goofy get bonuses
             goofy_get_bonus_location_pool = [self.multiworld.get_location(location, self.player) for location in
                                              Goofy_Checks.keys() if Goofy_Checks[location].yml != "Keyblade"]
+            if len(goofy_get_bonus_location_pool) > len(self.goofy_get_bonus_abilities):
+                raise Exception(f"Too little abilities to fill goofy get bonus locations for player {self.multiworld.get_file_safe_player_name(self.player)}.")
```

Comment:
Could use this instead
```suggestion
                raise Exception(f"Too little abilities to fill goofy get bonus locations for player {self.player_name}.")
```